dandiapi/api/management/commands/refresh_metadata.py

```diff
@@ -15,5 +15,6 @@ def refresh_metadata():
     for asset in Asset.objects.all():
         asset.save()
 
-    for version in Version.objects.all():
+    # Only update the metadata for draft versions
+    for version in Version.objects.filter(version='draft'):
```

Comment:
if that is the helper intended to be used to trigger update to `assetsSummary`, I do  not see a reason to go through all assets (and we do not even have GC yet, so it would be really all assets, right?) and mint new thousands of them if I get it right what this code does.

Reply:
It would not mint new assets, but you are right about doing more work than necessary. I added some arguments.